service/src/main/java/com/derpgroup/quip/metrics/Metrics.java

```diff
@@ -0,0 +1,121 @@
+package com.derpgroup.quip.metrics;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.derpgroup.quip.configuration.QuipConfig;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class Metrics {
+
+  private final static Logger LOG = LoggerFactory.getLogger(Metrics.class);
+
+  private static boolean initialized = false;
+  private static QuipMetrics metrics = new QuipMetrics();
+  private static String metricsFile;
+
+  private Metrics(){}
+
+  public static synchronized void init(QuipConfig config) throws IOException{
+    if(initialized){
+      throw new RuntimeException("Metrics is already initialized");
+    }
+    if(config==null){return;}
+    metricsFile = config.getMetricsFile();
+
+    try{
+      loadMetricsFromFile(metricsFile);
+    }catch(IOException e){
+      LOG.warn("Unable to load metrics from metrics file",e);
+    }
+
+    initialized = true;
+    LOG.info("Metrics initialized");
+  }
+
+  protected static QuipMetrics loadMetricsFromFile(String fileName) throws IOException{
```

Comment:
We should never be reading this back in - we don't need the data as part of the live service, and having this be bidirectional only makes it more complicated.